holidays/locale/en_US/LC_MESSAGES/AR.po

```diff
@@ -16,107 +16,138 @@ msgstr ""
 "Generated-By: pygettext.py 1.5\n"
 "X-Generator: Poedit 3.2.2\n"
 
-#: holidays/countries/argentina.py:51
+#. Special Bridge Holidays are given upto 3 days a year as long as it's declared
+#. 50 days before calendar year's end There's no Bridge Holidays declared in
+#. 2017
+#: holidays/countries/argentina.py:70
+msgid "Feriado con fines turísticos"
+msgstr "Bridge Public Holiday"
+
+#. Special Cases
+#: holidays/countries/argentina.py:73
 msgid "Bicentenario de la creación y primera jura de la bandera nacional"
 msgstr "Bicentenary of the creation and first oath of the national flag"
 
-#: holidays/countries/argentina.py:56
+#: holidays/countries/argentina.py:76
 msgid "Bicentenario de la Batalla de Tucumán"
 msgstr "Bicentenary of the Battle of Tucumán"
 
-#: holidays/countries/argentina.py:62
+#: holidays/countries/argentina.py:77
 msgid ""
 "Bicentenario de la sesión inaugural de la Asamblea Nacional Constituyente "
 "del año 1813"
 msgstr ""
-"Bicentenary of the inaugural session of the National Constituent Assembly "
-"of the year 1813"
+"Bicentenary of the inaugural session of the National Constituent Assembly of "
+"the year 1813"
 
-#: holidays/countries/argentina.py:67
+#: holidays/countries/argentina.py:81
 msgid "Bicentenario de la Batalla de Salta"
 msgstr "Bicentenary of the Battle of Salta"
 
-#: holidays/countries/argentina.py:81 holidays/countries/argentina.py:84
+#: holidays/countries/argentina.py:82
+msgid "Censo nacional 2022"
+msgstr "National Census Day 2022"
+
+#: holidays/countries/argentina.py:161 holidays/countries/argentina.py:164
 #, c-format
 msgid "%s (Observado)"
 msgstr "%s (Observed)"
 
-#. New Year's Day
-#: holidays/countries/argentina.py:90
+#. !!! New Year's Day !!! Status: In-Use
+#: holidays/countries/argentina.py:178
 msgid "Año Nuevo"
 msgstr "New Year's Day"
 
-#. Carnival days
-#: holidays/countries/argentina.py:94
+#: holidays/countries/argentina.py:185
 msgid "Día de Carnaval"
 msgstr "Carnival"
 
-#: holidays/countries/argentina.py:100
+#: holidays/countries/argentina.py:193
 msgid "Día Nacional de la Memoria por la Verdad y la Justicia"
 msgstr "Memory's National Day for the Truth and Justice"
 
-#. Good Friday
-#: holidays/countries/argentina.py:105
-msgid "Viernes Santo"
-msgstr "Good Friday"
-
-#: holidays/countries/argentina.py:109
+#. !!! Veterans Day and the Fallen in the Malvinas War !!! Status: In-Use
```

Comment:
It seems we need to use sort of convention here - to have an extra line after the comments describing the business logic/history and have a simple comment with English holiday name right on top of `self.tr` line.

I don't see any good reason why we would want to have this extra information in .po files.

Reply:
They're auto-generated via `make l10n` - I'll see if I can have them fixed for Thailand's case too.

Reply:
Yes, that's a part of l10n process - it adds comments for translators. 
You need to separate the unrelated comments from `self.tr`

Reply:
Fixed.